build/verify-archive.sh

```diff
@@ -10,8 +10,9 @@ set -euo pipefail
 apt-get update
 apt-get install -y cmake xz-utils
 
-tar xzf cockroach.src.tgz
-(cd cockroach-* && make install)
+workdir="$(mktemp -d)"
```

Comment:
They are unnecessary, but I do it out of habit for consistency. Don't feel strongly, so I dropped the quotes.